src/main/java/com/sarality/sync/SyncErrorLogger.java

```diff
@@ -0,0 +1,18 @@
+package com.sarality.sync;
+
+import hirondelle.date4j.DateTime;
+
+/**
+ * Log syncing errors
+ *
+ * @author satya (satya puniani)
+ */
+
+public interface SyncErrorLogger {
+
+  public void logError(Exception e);
+
+  public void logError(Exception e, Long errorRecordId, String errorTableName, DateTime errorTimeStamp,
```

Comment:
We should move timestamp and description out and then call teh others syncRecordId and syncTableName